src/postgraphql/cli.ts

```diff
@@ -39,9 +37,9 @@ program
   .option('-j, --dynamic-json', 'enable dynamic JSON in GraphQL inputs and outputs. uses stringified JSON by default')
   .option('-M, --disable-default-mutations', 'disable default mutations, mutation will only be possible through Postgres functions')
   .option('-l, --body-size-limit <string>', 'set the maximum size of JSON bodies that can be parsed (default 100kB) The size can be given as a human-readable string, such as \'200kB\' or \'5MB\' (case insensitive).')
-  .option('--secret <string>', 'DEPRECATED: See jwt.secret')
-  .option('-e, --jwt.secret <string>', 'the secret to be used when creating and verifying JWTs. if none is provided auth will be disabled')
-  .option('-A, --jwt.audience <list>', 'a comma separated list of audiences your jwt token can contain. If no audience is given the audience defaults to `postgraphql`', assign('audience', list), jwtOptions)
+  .option('secret <string>', 'DEPRECATED: Use jwt-secret instead')
+  .option('-e, --jwt-secret <string>', 'the secret to be used when creating and verifying JWTs. if none is provided auth will be disabled')
+  .option('-A, --jwt-audience <list>', 'a comma separated list of audiences your jwt token can contain. If no audience is given the audience defaults to `postgraphql`', (option: string) => option.split(','))
```

Comment:
Maybe instead of `<list>` here we just do `<string>`? I don’t people to mistakenly think that they have to put in a list 😣

We do this for `--schema` as well.